packages/ui/public/stellar.html

```diff
@@ -62,12 +62,30 @@
     </div>
   </div>
   <div class="nav-row flex flex-row items-center justify-between">
-    <div class="nav">
+    <div class="nav nav-switches flex flex-row">
       <a class="switch switch-solidity" href="/"><img src="/icons/solidity.svg" alt="solidity">Solidity</a>
       <a class="switch switch-cairo" href="/cairo"><img src="/icons/cairo.svg" alt="cairo">Cairo</a>
-      <a class="switch switch-stellar active" href="#"><img src="/icons/stellar.svg" alt="stellar">Stellar</a>
+      <a class="switch switch-stellar active" href="/stellar"><img src="/icons/stellar.svg" alt="stellar">Stellar</a>
```

Comment:
If this is `/stellar`, the clicking Stellar while on Stellar tab causes the page to refresh, which seems pointless.  Using `#` will cause it to do nothing, which is preferable.
```suggestion
      <a class="switch switch-stellar active" href="#"><img src="/icons/stellar.svg" alt="stellar">Stellar</a>
```